loky/backend/utils.py

```diff
@@ -1,7 +1,77 @@
+import os
+import sys
+import errno
+import signal
+import warnings
 import threading
+import subprocess
 
 
 def _flag_current_thread_clean_exit():
     """Put a ``_clean_exit`` flag on the current thread"""
     thread = threading.current_thread()
     thread._clean_exit = True
+
+
+def safe_terminate(process):
```

Comment:
I would rather rename this to a more explicit name such as `recursive_terminate`.